### ISSUE TITLE
Delivery options text does not have sufficient color contrast ratio

### ISSUE THREAD

**Issue by axamei** [pulibrary/requests#516]
"Library staff are working remotely to deliver materials digitally as long as electronic resources are available" does not have enough color contrast.
<img width="1196" alt="Screen Shot 2020-04-08 at 8 44 18 AM" src="https://user-images.githubusercontent.com/8161144/78786160-4e00d280-7976-11ea-9b7a-72496a7b2676.png">
